x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/rule_creation/common_flows.cy.ts

```diff
@@ -68,7 +70,8 @@ describe.skip('Common rule creation flows', { tags: ['@ess', '@serverless'] }, (
   it('Creates and enables a rule', function () {
     cy.log('Filling define section');
     importSavedQuery(this.timelineId);
-    // The following step is flaky due to a recent EUI upgrade. There is not currently a ticket for the underlying EUI issue, but when there is it will be linked on the test-failure ticket: https://github.com/elastic/kibana/issues/183437
+    // The following steps are flaky due to a recent EUI upgrade. There is not currently a ticket for the underlying EUI issue, but when there is it will be linked on the test-failure ticket: https://github.com/elastic/kibana/issues/183437
+    // fillRequiredFields();
```

Comment:
the issue refers to failing test on main and due to related integrations. 
Do required fields have the same problem?